src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -18,6 +19,11 @@
 short-summary: Manage Azure IoT Hub Device Provisioning Service.
 """
 
+helps['iot pnp'] = """
```

Comment:
You should call out customer facing changes (bug fixes, new commands) in history.rst. This excludes help text